projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -245,16 +245,7 @@ public function add_users_menu( $wp_admin = false ) { // phpcs:ignore VariableAn
 		add_submenu_page( 'users.php', esc_attr__( 'Add New', 'jetpack' ), __( 'Add New', 'jetpack' ), 'promote_users', 'https://wordpress.com/people/new/' . $this->domain, null, 1 );
 	}
 
-	/**
-	 * Adds Tools menu.
-	 *
-	 * @param bool $wp_admin_import Optional. Whether Import link should point to Calypso or wp-admin. Default false (Calypso).
-	 * @param bool $wp_admin_export Optional. Whether Export link should point to Calypso or wp-admin. Default false (Calypso).
-	 */
-	public function add_tools_menu( $wp_admin_import = false, $wp_admin_export = false ) {  // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-		// Export on Simple sites is always handled on Calypso.
-		parent::add_tools_menu( $wp_admin_import, false );
-	}
+	// add_tools_menu() - Inherit from Admin_Menu base class.
```

Comment:
I'd vote to leave the comment out. It seems odd to me to have a comment about why something isn't there; I'd expect that to be just more confusing to anyone other than the person that added `add_tools_menu()` here in the first place. The fact that `add_tools_menu` will run from the parent class should be self-evident too.